docs/wrappers/peer_lifecycle_chaincode_postscript.md

```diff
@@ -109,16 +109,19 @@ channel `mychannel`.
 ### peer lifecycle chaincode checkcommitreadiness example
 
 You can check whether a chaincode definition is ready to be committed using the
-``peer lifecycle chaincode checkcommitreadiness command, which will return
```

Comment:
it's not yours but we've been unnecessarily using double backticks in this file. would be nice to convert them to single backtick

Reply:
Thank you for having reviews.
New patch convert unnecessary double backticks to single backtick in the entire file.